src/test/storage/any_segment_iterable_test.cpp

```diff
@@ -83,6 +78,8 @@ INSTANTIATE_TEST_CASE_P(
     AnySegmentIterableTestInstances, AnySegmentIterableTest,
     ::testing::Values(SegmentEncodingSpec{EncodingType::Unencoded}, SegmentEncodingSpec{EncodingType::Dictionary},
                       SegmentEncodingSpec{EncodingType::Dictionary, VectorCompressionType::FixedSizeByteAligned},
-                      SegmentEncodingSpec{EncodingType::Dictionary, VectorCompressionType::SimdBp128}), );  // NOLINT
+                      SegmentEncodingSpec{EncodingType::Dictionary, VectorCompressionType::SimdBp128},
+                      SegmentEncodingSpec{EncodingType::FrameOfReference}, SegmentEncodingSpec{EncodingType::RunLength},
+                      SegmentEncodingSpec{EncodingType::LZ4}), );  // NOLINT
```

Comment:
Argh. We have too many places where these definitions have to be added. Can we centralize this? Maybe as part of a different PR?

Reply:
I do not really know what you're meaning. For testing, I rather like to explicitly list all variants and especially test the non-default vector compressions.

Do you mean you'd like a function that yields all possible combinations?

Reply:
LZ4 should have been added here when it was originally introduced. The fact that it wasn't shows that we have too many places where new segment types have to be added. I'll open an issue.

Reply:
Ah yes, got you. Makes sense.